sonic-psud/scripts/psud

```diff
@@ -606,6 +596,8 @@ class DaemonPsud(daemon_base.DaemonBase):
              (PSU_INFO_IN_CURRENT_FIELD, str(in_current)),
              (PSU_INFO_IN_VOLTAGE_FIELD, str(in_voltage)),
              (PSU_INFO_POWER_MAX_FIELD, str(max_power)),
+             (PSU_INFO_PRESENCE_FIELD, 'true' if _wrapper_get_psu_presence(index) else 'false'),
+             (PSU_INFO_STATUS_FIELD, 'true' if _wrapper_get_psu_status(index) else 'false'),
```

Comment:
the difference is not on the way of getting the psu data, but rather on the time sequence.
as we use the psu status to decide whether the psu is ok or not on in the utilities script, as long as we get this data at the latest time then it will prevent the mismatch fields problem